### DISCUSSION TITLE
Maybe update bug

### DISCUSSION THREAD

**Discussion by TatanAlternova** [supabase/supabase#1889] [Questions] [answered]
Hi supabase STAFF, I hope you have a nice day, this message is about an error using your database, I think might my error but I want to share with you for sure: I want to update email user from "users" table of supabase, the object data response say "email" and "new_email" let me understand the process is OK but this change never happent in "users" table.

print:
![image](https://user-images.githubusercontent.com/78916909/121284379-5e764b00-c8a2-11eb-85d4-bdf23a824e50.png)

update:
![image](https://user-images.githubusercontent.com/78916909/121284435-76e66580-c8a2-11eb-9b22-16529119e8b6.png)

info users:
![image](https://user-images.githubusercontent.com/78916909/121284474-84035480-c8a2-11eb-8750-75a661a7e82e.png)

**Comment by steve-chavez** [comment, marked as answer]
Hey @TatanAlternova,

That should be fine, supabase.auth uses [GoTrue](https://github.com/supabase/gotrue/), which adds some logic on top of the gable - see this [new_email](https://github.com/supabase/gotrue/blob/master/models/user.go#L39) change.
